src/physics/grid/PhysicsGridCalculator.i.hh

```diff
@@ -17,7 +17,7 @@ namespace celeritas
  * Calculate the cross section from the particle state.
  *
  * This signature is here to allow for potential acceleration by precalculating
- * log(E)
+ * log(E) as and operating on that instead of
```

Comment:
Go on... 😄

Reply:
Oh haha I just got this! 🐿️

Reply:
Yeah I have these all over the place in SCALE/Exnihilo from the days before we had regular MR reviews ;)